src/components/Accordion/index.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from "react"
+import { useColorMode, useColorModeValue } from "@chakra-ui/react"
+import {
+  Accordion as ChakraAccordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionPanel,
+  AccordionIcon,
+  Box,
+} from "@chakra-ui/react"
+
+const Accordion = () => {
+  const { toggleColorMode } = useColorMode()
+  const primary = useColorModeValue("#8282FF", "#FFE3D3")
```

Comment:
Adding to what Tyler said and replying to your original question, the color was fixed in figma file, we should use the semantic token [`primary.lowContrast`](https://github.com/ethereum/ethereum-org-website/blob/dev/src/@chakra-ui/gatsby-plugin/semanticTokens.ts#L42) in this case.

Reply:
Sure, I will fix this as soon as possible. Thanks for letting me know.